CMakeLists.txt

```diff
@@ -286,6 +286,12 @@ configure_file(
     @ONLY
 )
 
+configure_file(
```

Comment:
I think this is not needed? There are no variables for cmake to substitute in this file.